From 46fc9decadfa6b8c69c95f24a153dea2a0271e78 Mon Sep 17 00:00:00 2001
From: Helen Koike <helen.koike@collabora.com>
Date: Fri, 20 Oct 2023 10:40:51 -0300
Subject: [PATCH 650/834] ci/marge_queue: add missing python-dateutils to
 requirements.txt

marge_queue.py requires python-dateutils to run, so add it.

Signed-off-by: Helen Koike <helen.koike@collabora.com>
Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/25828>
---
 bin/ci/requirements.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/bin/ci/requirements.txt b/bin/ci/requirements.txt
index b8732b11667..6015acd56e8 100644
--- a/bin/ci/requirements.txt
+++ b/bin/ci/requirements.txt
@@ -2,6 +2,7 @@ aiohttp==3.8.3
 colorama==0.4.5
 filecache==0.81
 gql==3.4.0
+python-dateutil==2.8.2
 python-gitlab==3.5.0
 PyYAML==6.0
 ruamel.yaml.clib==0.2.7
-- 
2.42.0

